README.md

```diff
@@ -36,8 +36,61 @@ config :instrumental,
 ### Options
 
   * token (required) - api key for authenticating with instrumental
-  * host (optional) - host of instrumental collector
-  * port (optional) - port of instrumental collector
+  * host (optional) - host of instrumental collector, *default: collector.instrumentalapp.com*
+  * port (optional) - port of instrumental collector, *default: 8000*
+
+
+## Usage
+
+```elixir
+defmodule ImportantThing do
+  alias Instrumental, as: I
+
+  def compute_important_thing do
+    # tell Instrumental how long it takes to run a
+    # computation so we can know if our performance
+    # is changing over time.
+    value = I.time("ImportantThing.timing", &ImportantThing.compute/0)
```

Comment:
Actually, I think we already had this in the examples below this. So, two steps ahead of you, but I forgot by the time we talked about it!